docs/_may_milestone_.md

```diff
@@ -0,0 +1,62 @@
+## Monthly Milestone - May 2020
+
+### Overview
+
+### Goals
+
+- [ ] Migration Guide
+
+  - [ ] How to migrate LB3 components #3955 _depends on the spike result in
```

Comment:
the story numbers should exchange here
How to migrate LB3 components #5004 _depends on the spike result in #3955_

Reply:
hm. Not really. Supposedly #5004 will be done and it will shape the acceptance criteria for #3955 which we will start in May.

Reply:
I am afraid #5004 is very likely to slip :( I underestimated the amount of work needed 🙈

Reply:
Looks like I've mixed up the github issues. Fixed in the latest commit. Thanks.